rice-middleware/kim/kim-impl/src/main/java/org/kuali/rice/kim/impl/group/GroupServiceImpl.java

```diff
@@ -92,6 +92,36 @@ public Group getGroup(String groupId) throws RiceIllegalArgumentException {
     }
 
     @Override
+    public List<Group> getParentGroupsByGroupsIds(List<String> groupIds) {
+        return getParentGroupsByGroupsIds(groupIds, Collections.EMPTY_SET);
```

Comment:
This method should also do some sanity checking like the `getParentGroups` method.